src/modules/poll/index.js

```diff
@@ -0,0 +1,317 @@
+
+const { hri } = require('human-readable-ids');
+const moment = require('moment');
+const isEqual = require('lodash/isEqual');
+const command = require('../command');
+
+const addPoll = async function (response, {
+    getModuleData,
+    updateModuleData,
+    id,
+    i18n,
+}) {
+    const { args: { question, options } } = response;
+    const { pollsList = [] } = await getModuleData('poll');
+    const pollId = hri.random();
+    const newPoll = {
+        authorId: id,
+        isOpen: true,
+        question,
+        options,
+        pollId,
+        dateCreated: new Date(),
+    };
+
+    updateModuleData('poll', {
```

Comment:
Offtop: Imo, it was easier to make it as an object.
Now you have:
```js
poll: {
  pollList: [
    {pollId: id, ...someData},
    {pollId: otherId, ...otherData}
  ]
}
```
My option:
```js
poll: {
  pollList: {
    id: {...someData},
    otherId: {...otherData}
  }
}
```

Reply:
No need to rework